src/util/classes.ts

```diff
@@ -42,12 +47,26 @@ export const defaultCategories: Category[] = [
   { name: ['Media', 'Video'], rule: { type: 'regex', regex: 'YouTube|Plex|VLC' } },
   {
     name: ['Media', 'Social Media'],
-    rule: { type: 'regex', regex: 'reddit|Facebook|Twitter|Instagram|devRant', ignore_case: true },
+    rule: {
+      type: 'regex',
+      regex: 'reddit|Facebook|Twitter|Instagram|devRant',
+      ignore_case: true,
```

Comment:
An issue I had with this PR was that it had `ignore_case` set to true (which I think should be avoided in the defaults), but I now notice that was the case before anyway, so not really an issue for getting it merged.